src/app/styles/index.postcss.css

```diff
@@ -122,6 +122,10 @@ article table {
   @apply border border-sc-gray-3 table-auto w-full mb-4 text-sc-text-4 text-sc-gray-1;
 }
 
+article .table-wrapper {
```

Comment:
The `table-wrapper` identifier is too generic IMHO. It doesn't convey the idea its content is allowed to overflow horizontally. If we end up having multiple configurations for wrapping enclosed tables, we'll hardly differentiate them if the first one is simply named "wrapper".

I suggest to name it after the behavior it introduces: `overflow-horizontal-content`, or `overflow-horizontal-content-for-XXX` where XXX relates to `w-[95vw]`.

Reply:
What if we apply this to all table inside article ? It simplify things

Reply:
_Disclaimer: I haven't browsed the current source code. Please comment, and disregard my suggestion if it doesn't follow the current code style, or convention._

This is my suggestion, indeed, so it doesn't add one level of indirection for the reader before understanding what it is. The class name `table-wrapper` gives me the place in the HTML code, however at the moment I read it I'm already in the HTML code. It gives me the identifier to look for in the CSS, though this is the indirection I was mentioning.

Reply:
@btrd What do you think about this solution and about the class naming ?